scene/2d/line_2d.cpp

```diff
@@ -95,6 +95,26 @@ float Line2D::get_width() const {
 	return _width;
 }
 
+void Line2D::set_curve(const Ref<Curve> &curve) {
+	// Cleanup previous connection if any
+	if (_curve.is_valid()) {
+		(**_curve).disconnect(CoreStringNames::get_singleton()->changed, this, "_curve_changed");
```

Comment:
Can't you use `_curve->disconnect()` directly?

Reply:
the _gradient member is handled the same way, so I just mimicked what was previously done for uniformity.
Should I change the set_gradient as well ?

Reply:
If it works, yes, I didn't check that it would actually compile. It's just that it's the first time I see this construct in Godot's codebase, while `Ref<>` is used everywhere.

Reply:
Done, I changed it for _gradient, _curve and _texture.